omada-controller.json

```diff
@@ -0,0 +1,76 @@
+{
+    "TP-Link Omada": {
+        "containers": {
+            "omada-controller": {
+                "image": "mbentley/omada-controller",
+                "launch_order": 1,
+                "ports": {
+                    "8088": {
+                        "description": "Management HTTP port",
+                        "host_default": 8088,
+                        "label": "Redirect to WebUI",
+                        "protocol": "tcp",
+                        "ui": true
+                    },
+                    "8043": {
+                        "description": "Management HTTPS port",
+                        "host_default": 8043,
+                        "label": "HTTPS WebUI",
+                        "protocol": "tcp",
+                        "ui": false
+                    },
+                    "8843": {
+                        "description": "Portal HTTPS port",
+                        "host_default": 8843,
+                        "label": "Portal HTTPS port",
+                        "protocol": "tcp",
+                        "ui": false
+                    },
+                    "29814": {
+                        "description": "Adopt & Manage Omada devices",
+                        "host_default": 29814,
+                        "label": "Adopt & Manage Omada devices",
+                        "protocol": "tcp",
+                        "ui": false
+                    },
+                    "29810": {
+                        "description": "Discover Omada devices",
+                        "host_default": 29810,
+                        "label": "Discover Omada devices",
+                        "protocol": "udp",
+                        "ui": false
+                    }
+                },
+                "volumes": {
+                    "/opt/tplink/EAPController/data": {
+                        "description": "Choose a DATA Share for Omada Controller files. Eg: create a Share called omada-controller-data for this purpose alone.",
+                        "label": "EAPController DATA Storage"
+                    },
+                    "/opt/tplink/EAPController/work": {
+                        "description": "Choose a WORK Share for Omada Controller files. Eg: create a Share called omada-controller-work for this purpose alone.",
+                        "label": "EAPController WORK Storage"
+                    },
+                    "/opt/tplink/EAPController/logs": {
+                        "description": "Choose a LOGS Share for Omada Controller files. Eg: create a Share called omada-controller-logs for this purpose alone.",
+                        "label": "EAPController LOGS Storage"
+                    }
+                },
+                "environment": {
+                    "TZ": {
+                    "description": "Timezone for the controller.",
+                    "label": "TZ",
+                    "index": 1	
+                    }
+                }
```

Comment:
The timezone mapping is not really required. The Rockon helper already maps those to the TZ of the Rockstor appliance
```suggestion

```